inc/class-multitaxo-plugin.php

```diff
@@ -201,6 +201,7 @@ public function create_database_tables() {
 	 * @return void
 	 */
 	public function delete_database_tables() {
```

Comment:
Maybe we shoudl remove this empty function instead of adding comments to make it pass the tests ;-)